app/app.js

```diff
@@ -3,7 +3,9 @@
     angular.module("app", [
         "ngRoute", 
         "chatApp.DataModel",
+        "chatApp.UserSession",
```

Comment:
Please, do not split application into such modules.
Module should include services, controllers, directives etc.
For our task it is enough to have 1 module called "app"